Sprintfinity3902/Dungeon/RoomLoader.cs

```diff
@@ -47,13 +47,15 @@ public class RoomLoader : IRoomLoader
         public IDoor DoorRight { get; set; }
         int enemyID;
         int spikeNum;
+
+        private IPlayer link;
         
 
         // Have this input a filename and then load the room.
-        public RoomLoader(IRoom room)
+        public RoomLoader(IPlayer player)
         {
             // Really think there is a better way to list these files, just a demo for the time being though.
-            Initialize(room);
+            link = player;
         }
```

Comment:
This looks like a big change to the RoomLoader that I'm not quite sure I understand.

Reply:
I get the the hand needs to know link's location, but does the Initialize still occur?